db/patch74.sql

```diff
@@ -0,0 +1,5 @@
+# adding the bio column to the user info
+
+ALTER TABLE `user` add `biography` VARCHAR(400);
```

Comment:
Wouldn't a column type of `TEXT` be better here?. That way we don't add those 400*4 byte to the max row-length :wink:

Reply:
agree here. text would be nice.

Reply:
Thanks for the review. I've changed the data type to `TEXT`